tests/ci_build/test_jvm_cross.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+set -x
+
+# Initialize local Maven repository
+./tests/ci_build/initialize_maven.sh
+
+# Get version number of XGBoost4J and other auxiliary information
+cd jvm-packages
+xgboost4j_version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)
+maven_compiler_source=$(mvn help:evaluate -Dexpression=maven.compiler.source -q -DforceStdout)
+maven_compiler_target=$(mvn help:evaluate -Dexpression=maven.compiler.target -q -DforceStdout)
+spark_version=$(mvn help:evaluate -Dexpression=spark.version -q -DforceStdout)
+scala_version=$(mvn help:evaluate -Dexpression=scala.version -q -DforceStdout)
+scala_binary_version=$(mvn help:evaluate -Dexpression=scala.binary.version -q -DforceStdout)
+
+# Install XGBoost4J JAR into local Maven repository
```

Comment:
hmmm....did you meet any problem when directly using mvn install?

Reply:
Note that the build is happening in the previous stage. Only the JAR files are copied over